cdap-watchdog/src/main/java/co/cask/cdap/metrics/store/DefaultMetricDatasetFactory.java

```diff
@@ -59,16 +57,10 @@
   private final DatasetFramework dsFramework;
 
   @Inject
-  public DefaultMetricDatasetFactory(final CConfiguration cConf, final DatasetFramework dsFramework) {
-    this(dsFramework, cConf);
```

Comment:
Previously we were wrapping dsFramework into NamespacedDatasetFramework, which is why we had private constructor, however wrapping got removed in this commit - https://github.com/caskdata/cdap/commit/6e9a6654d40879a14a60313270321cb6b8b9eb8d#diff-7cf678edd77bdb39c8ce63abdb509246L57

So we don't need private constructor anymore.